PWGCF/Flow/CMakeLists.txt

```diff
@@ -12,3 +12,4 @@
 add_subdirectory(Core)
 add_subdirectory(Tasks)
 add_subdirectory(TableProducer)
+add_subdirectory(DataModel)
```

Comment:
This file should not be affected

Reply:
Hi Victor, I have restored all the PWGCF files to origin/master